Connected web3 provider at 10.195.2.134:22006
Date.now(): 1580298891053 startTime: 1580299015806 delay: 124756
Deploying smart contract from 0xf19bd883f468433d2a50910be0aa8d3b5e56b95e
TX HASH:
 0xc041c629398664776c8716b298ee75960c8bf9d952d1e0b8eeb35c6fa179ee67
RECEIPT:
 {
  blockHash: '0x07b18dd9aabcd863217bfebe4ce47cc4abbfe164a92a4ea8b0bbe1f2a256d9c2',
  blockNumber: 5,
  contractAddress: '0x537dcb383Bb33a37714f0A0abA09fadD47301B42',
  cumulativeGasUsed: 218495,
  from: '0xf19bd883f468433d2a50910be0aa8d3b5e56b95e',
  gasUsed: 218495,
  logs: [],
  logsBloom: '0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
  status: true,
  to: null,
  transactionHash: '0xc041c629398664776c8716b298ee75960c8bf9d952d1e0b8eeb35c6fa179ee67',
  transactionIndex: 0
}
Contract at: 0x537dcb383Bb33a37714f0A0abA09fadD47301B42

Evaluating  8000txs...

Sending TXS started at 1580299015946 ...
startingBlock: 21
... Sending TXS finished at 1580299123507
Total send duration: 107561
... Waiting for mining process to finish! ...
(node:14791) UnhandledPromiseRejectionWarning: Error: Failed to check for transaction receipt:
{}
    at Object._fireError (/home/ubuntu/benchprezz/node_modules/web3-core-method/node_modules/web3-utils/src/index.js:56:17)
    at /home/ubuntu/benchprezz/node_modules/web3-core-method/src/index.js:260:23
    at runMicrotasks (<anonymous>)
    at runNextTicks (internal/process/task_queues.js:58:5)
    at listOnTimeout (internal/timers.js:501:9)
    at processTimers (internal/timers.js:475:7)
(node:14791) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:14791) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
ERROR:
 Error: Failed to check for transaction receipt:
{}
    at Object._fireError (/home/ubuntu/benchprezz/node_modules/web3-core-method/node_modules/web3-utils/src/index.js:56:17)
    at /home/ubuntu/benchprezz/node_modules/web3-core-method/src/index.js:260:23
    at runMicrotasks (<anonymous>)
    at runNextTicks (internal/process/task_queues.js:58:5)
    at listOnTimeout (internal/timers.js:501:9)
    at processTimers (internal/timers.js:475:7)
========================================================
========================================================

/home/ubuntu/benchprezz/benchmark/tpsUnsigned.js:113
                throw 'Blockchain capabilities EXCEEDED - ABORTING test run!';
                ^
Blockchain capabilities EXCEEDED - ABORTING test run!
